frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/DeleteModal/DeleteModal.tsx

```diff
@@ -19,29 +21,39 @@ export type DeleteModalProps = {
  *
  * @property {boolean}[isOpen] - If the modal is open or not
  * @property {function}[onClose] - Function to execute on close
- * @property {function}[onDelete] - Function to execute on click delete
- * @property {string}[appName] - The name of the app to delete changes on
+ * @property {string}[app] - The app
+ * @property {string}[org] - The org
  *
  * @returns {ReactNode} - The rendered component
  */
-export const DeleteModal = ({
-  isOpen,
-  onClose,
-  onDelete,
-  appName,
-}: DeleteModalProps): ReactNode => {
+export const DeleteModal = ({ isOpen, onClose, app, org }: DeleteModalProps): ReactNode => {
```

Comment:
I've noticed that the code remains unchanged in this pull request. However, for increased specificity and accuracy, it would be advisable to modify the return type to JSX.Element. :) 

```suggestion
export const DeleteModal = ({ isOpen, onClose, app, org }: DeleteModalProps): JSX.Element => {
```